examples/nextjs/src/app/[[...slug]]/page.js

```diff
@@ -26,33 +26,71 @@ export async function generateMetadata({ params, searchParams }) {
         const title = page?.friendlyName || page?.title;
 
         return {
-            title,
+            title
         };
     } catch (e) {
         return {
-            title: "not found",
+            title: 'not found'
         };
     }
 }
 
 export default async function Home({ searchParams, params }) {
-    const getPageData = async () => {
-        const path = params?.slug?.join("/") || "/";
+    const getNavigation = async () => {
+        const path = params?.slug?.join('/') || '/';
         const pageParams = getPageRequestParams({
             path,
-            params: searchParams,
+            params: searchParams
         });
 
-        const { pageAsset, error: pageError } = await fetchPageData(pageParams);
         const { nav, error: navError } = await fetchNavData(pageParams.language_id);
 
         return {
             nav,
-            pageAsset,
-            error: pageError || navError,
+            error: navError
         };
     };
-    const { pageAsset, nav, error } = await getPageData();
+    const { nav, error } = await getNavigation();
+
+    const data = await client.gql({
+        page: {
+            url: params?.slug?.join('/'),
+            language: searchParams?.language_id,
+            mode: 'EDIT_MODE',
```

Comment:
We need to control this, should be optional for them.

Reply:
agree